.github/workflows/nodejs.yml

```diff
@@ -11,40 +11,45 @@ on:
       - master
 
 jobs:
-  build:
+  test:
```

Comment:
The job is renamed to test, but it includes semantic release. Is this unintentional leftover from the experimentation? 
It seems to work fine, just the name is not indicative of the behavior. We can rename it back to something generic or split into multiple jobs for build, test, and release with dependencies on each other. Thoughts?

Reply:
Correct,.. I think splitting will be optimum for this scope, as we may add new jobs in the future..

I think @BKristenssonAlfsson  may have some thoughts

Reply:
I think like @sniperadmin says, better to have one clear test job and one clear build job.